apps/judicial-system/backend/src/app/modules/file/file.service.ts

```diff
@@ -220,22 +215,22 @@ export class FileService {
       courtCaseNumber,
     )
 
-    const documentId = await this.throttle
+    await this.throttle
 
-    const s3Key = file.key
-
-    const [nrOfRowsUpdated] = await this.fileModel.update(
-      { state: CaseFileState.STORED_IN_COURT, key: documentId },
```

Comment:
So are we saying that we keep the same `case_file.key` after uploading to court? No need to get the `documentId` anymore?

Reply:
Yes, we have never used the documentId we get from Auður, so I decided to just drop it.